marge/batch_job.py

```diff
@@ -89,7 +89,7 @@ def get_mergeable_mrs(self, merge_requests):
             except CannotMerge as ex:
                 log.warning('Skipping unmergeable MR: "%s"', ex)
                 self.unassign_from_mr(merge_request)
-                merge_request.comment("I couldn't merge this branch: {}".format(ex))
+                merge_request.comment("I couldn't merge this MR: {}".format(ex))
```

Comment:
Hmm... is this part of the change good or bad? :thinking: Speaking a bit more low-level, it is actually a _branch_ that gets merged. The MR is an abstraction layer on top of the "branch" concept (which is a part of the core `git` terminology).

Is adopting the "abstraction-layer" language here an improvement or not? :thinking: I'm not fully convinced that it is, but please give me your best arguments for why you think this deserves to be changed.

Reply:
![image](https://user-images.githubusercontent.com/1417619/210766980-62a096c1-c287-43f3-a00c-46841543c860.png)

vs

![image](https://user-images.githubusercontent.com/1417619/210767003-241010d8-e227-4608-885b-c1880d295bbe.png)

We always talk about "approve my MR", "can you merge your MR", etc etc. No one talks about "merge your branch".

Reply:
Also, it's the _MR_ that cannot be merged because the approvals are missing. There's no approval concept in core `git` :grin:

Reply:
Fair enough, even though I guess you _could_ say that the approval concept in core `git` would be "email LKML your patches and see if Linus will import them into his tree". :joy: 

I approved the PR now with the suggestion to spell out _merge request_ instead of abbreviating it. I think it makes more sense that way or WDYT? Reads better, at least to me.

Reply:
:heavy_check_mark: